compact/far_forward/ZDC.xml

```diff
@@ -11,11 +8,10 @@
       Zero Degree Calorimeter General Parameters
       -------------------------------------------
     </comment>
-    <constant name="ZDC_z_pos"         value="3560.0 * cm"/> <!-- need to shift FORWARD toward IP to account for 50cm shift -->
-    <constant name="ZDC_x_pos"         value="ZDC_z_pos * ionCrossingAngle"/>
+    <constant name="ZDC_r_pos"         value="3550.0 * cm"/> <!-- Make it match to the CAD figure 22/Nov/22 -->
     <constant name="ZDC_y_pos"         value="0.0 * cm"/>
     <constant name="ZDC_rotateX_angle" value="0.0 * rad"/>
-    <constant name="ZDC_rotateY_angle" value="ionCrossingAngle"/>
+    <constant name="ZDC_rotateY_angle" value="-1.* ionCrossingAngle"/>
```

Comment:
This doesn't really address the issue I pointed out. It just moves the minus sign somewhere else. Why is a minus sign needed at all?

Reply:
Due to historical reason, local rotation of volumes in the detector templates (cpp files) under src/ is defined as RotationZYX(rot.z(), -rot.y(), rot.x()) in all the ZDC related cpp files (including the ATHENA ones). Current -1 factor for the rotation in the ZDC.xml file gives proper alignment. 
I will check the situation for ATHENA ZDC as it could have wrong alignment as well. 
I and/or Po-Ju (@pjlin) will reverse the rotation in cpp files of both ECCE and ATHENA type and take out -1 factors from XML file during this week.